css/style.css

```diff
@@ -17,3 +17,222 @@ body {
  */
 
 
+.bar {
+    width: 100%;
+    height: 70px;
+    display: flex;
+    background-color: white;
+    position: fixed;
+    z-index: 100;
+}
+
+.pic {
+    justify-content: center;
+    width: 30%;
+    display: flex;
+}
+
+.logo {
+width: 35px;
+height: 70px;
+margin-left: 30%; 
```

Comment:
I suggest using "em" unit instead of percentage for margins , it would make it much easier when you are going to make the website responsive to work on different devices.